File: writer.mjs
import { writeFileSync, mkdirSync } from "fs";
import { dirname, join } from "path";
const base = "/Users/designs/Downloads/Independency day/indochine/indochine-nextjs/src";
function w(rel, b64) { const fp = join(base, rel); mkdirSync(dirname(fp), { recursive: true }); writeFileSync(fp, Buffer.from(b64, "base64").toString()); console.log("Written: " + rel); }

File: writer.py

import os

base = '/Users/designs/Downloads/Independency day/indochine/indochine-nextjs/src'

def w(rel, content):
    fp = os.path.join(base, rel)
    os.makedirs(os.path.dirname(fp), exist_ok=True)
    with open(fp, 'w') as f:
        f.write(content)
    print('Written: ' + rel)

print('Script loaded, writing files...')
